packages/fakta-beregning-redesign/package.json

```diff
@@ -29,6 +29,7 @@
     "@navikt/ds-react": "7.1.0",
     "@navikt/ft-form-hooks": "^5.1.1",
     "@navikt/ft-form-validators": "^2.4.1",
+    "@navikt/ft-frontend-saksbehandling-internal": "workspace:^",
```

Comment:
Du bør endra navn til noko som skildrar pakka funksjonelt. Huksa at det kan vera (og er) fleire interne pakker i repoet

Reply:
Jeg kan endre til internal-components eller noe sånt? Men hvis vi får behov for noe som helst annet enn komponenter må vi lage en ny pakke fordi navnet blir feil 😛